Testing System/src/main/webapp/pages/add.jsp

```diff
@@ -25,7 +25,7 @@
     </div>
 
     <div class="container signin">
-        <p>Already have an account? <a href="#">Sign in</a>.</p>
+        <p>Already have an account? <a href="/singin">Sign in</a>.</p>
```

Comment:
Please implement the POST method for Sign-in servlet or put a mock-up (fake until you make it).